apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -25,6 +26,15 @@ const buttonStoryMeta: ButtonStoryMeta = {
     title: 'Button',
     component: 'pie-button',
     argTypes: {
+        tag: {
+            description: 'Choose the HTML element that will be used to render the button.<br>For this story, the prop has the value of `button`. See the Anchor story to interact with the component when this prop has a value of `a`.',
```

Comment:
We probably don't want this description on the Anchor story (which is what currently happens)
<img width="1722" alt="Screenshot 2024-08-30 at 14 36 24" src="https://github.com/user-attachments/assets/4cbcf5a4-b264-43c0-97cd-fb86f8f305be">